render/filters.go

```diff
@@ -73,25 +73,24 @@ func ComposeFilterFuncs(fs ...FilterFunc) FilterFunc {
 // Filter removes nodes from a view based on a predicate.
 type Filter struct {
 	Renderer
-	FilterFunc   FilterFunc
-	filterPseudo bool
```

Comment:
@2opremio had a go and its equally pervasive.  The problem is changing the default filter to not ignore pseudo nodes pushes that into every calling site, for the sake of these few places where we want to not ignore pseudo node.  

I did manage to move some code into tests though.